### PR TITLE
Restored updated dreamsnaps

### PR DESCRIPTION
**Pull request by sndrs**
cc @crifmulholland @Pearson82 

dreamsnaps 2
dreamsnaps 2 mobile
http://www.browserstack.com/screenshots/3b28df35d7d671d6a75c39179a0534237db3cda8

### CODE REVIEW

**Comment by Pearson82** [comment]
Hey,
Were we not intending to have the 'Latest from' bit before the name of the tag? @sndrs

File: static/src/stylesheets/module/facia/item-tones/_tone-news.scss [hunk 1]
```diff
@@ -18,6 +18,10 @@
     .flyer__arrow-icon {
         fill: colour(news-default);
     }
+
+    .fc-item__kicker--dreamsnap:before {
+        color: colour(neutral-1);
+    }
```
Comment (ironsidevsquincy) [inline static/src/stylesheets/module/facia/item-tones/_tone-news.scss]: can these now the be standard and live in `_fc-item--list.scss`

File: common/app/views/fragments/items/elements/facia_cards/kicker.scala.html [hunk 2]
```diff
@@ -1,51 +1,51 @@
-@(header: layout.FaciaCardHeader)(implicit request: RequestHeader)
+@(header: layout.FaciaCardHeader, classes: String = "")(implicit request: RequestHeader)
```
Comment (ironsidevsquincy) [inline common/app/views/fragments/items/elements/facia_cards/kicker.scala.html]: might be nicer if `classes` is a `Seq`

**Comment by sndrs** [comment]
@Pearson82 no, editorial didn't want them (they're not always appropriate)

**Comment by Pearson82** [comment]
@sndrs <https://github.com/sndrs>  Ah, ok. I thought that was the whole
point of these?!  [?]

Could you maybe just take the text size of the label down a touch on some
of the smaller sizes. It looks like the label competes with the headline a
bit. I think the headline should always be bigger than the label.

That make sense?


On 8 January 2015 at 16:17, Alex Sanders <notifications@github.com> wrote:

> @Pearson82 <https://github.com/Pearson82> no, editorial didn't want them
> (they're not always appropriate)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/guardian/frontend/pull/7697#issuecomment-69203118>.
>

-- 

------------------------------
Visit theguardian.com. On your mobile and tablet, download the Guardian 
iPhone and Android apps theguardian.com/guardianapp and our tablet editions 
theguardian.com/editions.  Save up to 57% by subscribing to the Guardian 
and Observer - choose the papers you want and get full digital access. 
 Visit subscribe.theguardian.com

This e-mail and all attachments are confidential and may also be 
privileged. If you are not the named recipient, please notify the sender 
and delete the e-mail and all attachments immediately. Do not disclose the 
contents to another person. You may not use the information for any 
purpose, or store, or copy, it in any way.  Guardian News & Media Limited 
is not liable for any computer viruses or other material transmitted with 
or as part of this e-mail. You should employ virus checking software.
 
Guardian News & Media Limited is a member of Guardian Media Group plc. Registered 
Office: PO Box 68164, Kings Place, 90 York Way, London, N1P 2AP.  Registered 
in England Number 908396

**Comment by sndrs** [comment]
@Pearson82 
screen shot 2015-01-08 at 17 23 56

**Comment by Pearson82** [comment]
Yeah that works better for me.

Cheers

@sndrs

On 8 January 2015 at 17:25, Alex Sanders <notifications@github.com> wrote:

> @Pearson82 <https://github.com/Pearson82>
> [image: screen shot 2015-01-08 at 17 23 56]
> <https://cloud.githubusercontent.com/assets/867233/5666910/4b4830ae-975b-11e4-9130-00ebf050da4a.png>
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/guardian/frontend/pull/7697#issuecomment-69214851>.
>

-- 

------------------------------
Visit theguardian.com. On your mobile and tablet, download the Guardian 
iPhone and Android apps theguardian.com/guardianapp and our tablet editions 
theguardian.com/editions.  Save up to 57% by subscribing to the Guardian 
and Observer - choose the papers you want and get full digital access. 
 Visit subscribe.theguardian.com

This e-mail and all attachments are confidential and may also be 
privileged. If you are not the named recipient, please notify the sender 
and delete the e-mail and all attachments immediately. Do not disclose the 
contents to another person. You may not use the information for any 
purpose, or store, or copy, it in any way.  Guardian News & Media Limited 
is not liable for any computer viruses or other material transmitted with 
or as part of this e-mail. You should employ virus checking software.
 
Guardian News & Media Limited is a member of Guardian Media Group plc. Registered 
Office: PO Box 68164, Kings Place, 90 York Way, London, N1P 2AP.  Registered 
in England Number 908396

**Comment by ironsidevsquincy** [comment]
:+1:

**Pull request merged**